### DISCUSSION TITLE
data binding and state management

### DISCUSSION THREAD

**Discussion by k11q** [pixijs/pixijs#9004] [Q&A] [answered]
Hi I come from vue where data is reactive and 2 way binding is easy with model. Is there a way I can somehow bind values for example the width of sprite to a dynamic json data, so whenever I change the json the width change too?

Thank you

**Comment by miltoncandelero** [comment, marked as answer]
out of the box? nope

You might look into things like `ObservablePoint`, that calls a callback whenever is modified, but that's as close to _reactive_ as pixi gets 😅
